app/frontend/src/features/messages/groupchats/CreateGroupChat.tsx

```diff
@@ -0,0 +1,107 @@
+import { BoxProps } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { Autocomplete } from "@material-ui/lab";
+import React, { useEffect, useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import Alert from "../../../components/Alert";
+import Button from "../../../components/Button";
+import CircularProgress from "../../../components/CircularProgress";
+import TextField from "../../../components/TextField";
+import { User } from "../../../pb/api_pb";
+import { service } from "../../../service";
+import { useAppDispatch, useTypedSelector } from "../../../store";
+import { fetchUsers, getUsers } from "../../userCache";
+
+const useStyles = makeStyles({ root: {} });
+
+export interface GroupChatListProps extends BoxProps {
+  createGroupChat: (title: string, users: User.AsObject[]) => void;
```

Comment:
```suggestion
  createGroupChat: (title: string, users: User.AsObject[]) => Promise<void>;
```
As this is meant to be an async function?